pkg/streamingpromql/types/operator.go

```diff
@@ -10,6 +10,14 @@ import (
 	"github.com/prometheus/prometheus/promql/parser/posrange"
 )
 
+type PrepareParams struct {
+	QueryStats *QueryStats
+}
+
+func (p *PrepareParams) Clone() PrepareParams {
+	return PrepareParams{}
```

Comment:
Shouldn't this set `QueryStats`?

Reply:
Oh, it's just a leftover, I decided not to use Clone() at all, keeping things simple for now,  thanks for catching